src/util/piped_process.cpp

```diff
@@ -30,8 +101,127 @@
 
 piped_processt::piped_processt(const std::vector<std::string> commandvec)
 {
+  // Default state
```

Comment:
Would it be possible to split up this mega-function a bit?

Reply:
Split out a small helper function, but the function is pretty straightforward, just many lines due to many arguments to function calls to create Windows processes/pipes (also lots of commends/explanation).